GoogleSignIn/Sources/GIDToken.m

```diff
@@ -55,4 +57,39 @@ - (void)encodeWithCoder:(NSCoder *)encoder {
   [encoder encodeObject:_expirationDate forKey:kExpirationDateKey];
 }
 
+#pragma mark - isEqual
+
+- (BOOL)isEqual:(id)object {
+  if (self == object) {
+    return YES;
+  }
+  if (![object isKindOfClass:[GIDToken class]]) {
```

Comment:
For clarity, you might take the nil handling out of this line and add it in a new condition at the top of the method:

```
if (object == nil) {
  return NO;
}
```

Reply:
Done.